### PR TITLE
add cephfs thinruntime

### PR DESCRIPTION
**Pull request by baowj-678**
### Ⅰ. Describe what this PR does

add cephfs thinruntime.

### Ⅳ. Describe how to verify it

see readme.

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: addons/cephfs/docker/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+set +x
+
+docker build . --network=host -f Dockerfile -t baowj/cephfs:v0.1
+
+docker push baowj/cephfs:v0.1
```
Comment (cheyang) [inline addons/cephfs/docker/build.sh:6]: Add new line pls.

File: addons/cephfs/readme.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  name: nginx
+spec:
+  containers:
+    - name: nginx
+      image: nginx
+      volumeMounts:
+        - mountPath: /data
+          name: nfs-demo
+  volumes:
+    - name: nfs-demo
```
Comment (cheyang) [inline addons/cephfs/readme.md:61]: Why using the name `nfs` ?

File: addons/cephfs/docker/build.sh (hunk 1 above)
Comment (baowj-678) [reply addons/cephfs/docker/build.sh:6]: ok, I will add a new line.

File: addons/cephfs/readme.md (hunk 2 above)
Comment (baowj-678) [reply addons/cephfs/readme.md:61]: Oh, I just followed the nfs doc and forgot to change it. I will change it.

**Review by cheyang — APPROVED** [review]
/lgtm
/approve

**Pull request merged**